### PR TITLE
account_cutoff_base and prepaid adapted for operating unit

### PR DESCRIPTION
**Pull request by whulshof**
account_cutoff_base and -prepaid modules are not working in Operating Unit environment if the choice is for balanced BS/PL per OU.
This PR brings a solution. Also the generation of cutoff lines is rewritten in SQL, otherwise it could take too long for a large number of accruals.

### CODE REVIEW

File: account_cutoff_base_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Cut-off Base Operating Unit',
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'license': 'AGPL-3',
+    'summary': 'OU adaptation for Base module Account Cut-offs',
+    'author': 'Magnus,Odoo Community Association (OCA)',
+    'website': 'http://www.magnus.nl',
```
Comment (nikul-serpentcs) [inline account_cutoff_base_operating_unit/__manifest__.py:13]: @whulshof URL Should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: account_cutoff_prepaid_operating_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Cut-off Prepaid OU',
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'license': 'AGPL-3',
+    'summary': 'OU adaptation Prepaid Expense, Prepaid Revenue',
+    'author': 'Magnus,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```
Comment (nikul-serpentcs) [inline account_cutoff_prepaid_operating_unit/__manifest__.py:13]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by AaronHForgeFlow** [comment]
something wrong in the commit history there are commits from other modules

**Pull request closed without merging**